rom/laghos.cpp

```diff
@@ -163,6 +164,8 @@ int main(int argc, char *argv[])
     Array<double> twep;
     Array2D<int> twparam;
     ROM_Options romOptions;
+    int nnlsWindow0 = -1;
```

Comment:
Maybe we need a documentation for these new ideas, e.g., in the Overleaf document.